share/completions/blender.fish

```diff
@@ -0,0 +1,125 @@
+function __blender_list_scenes -a file
+    stdbuf --output=0 blender --background test.blend --python-expr 'import bpy
+
+for name in [scene.name for scene in list(bpy.data.scenes)]:
+    print(name)' | head --lines=-2 | tail --lines=+6
```

Comment:
These head and tail numbers seem a bit arbitrary. What are they needed for?

Reply:
Without these numbers there are extra unneded output:
```
Color management: using fallback mode for management
Color management: Error could not find role data role.
Blender 3.0.1
Read prefs: /home/emilyseville7cfg/.config/blender/3.0/config/userpref.blend
Color management: scene view "Filmic" not found, setting default "Standard".
Read blend: /home/emilyseville7cfg/Documents/open-source/other/forks/fish-shell/test.blend
Scene
Scene.001

Blender quit
```

Everything is being put in stdout, so I can't just discard anything in an easy way.

Reply:
Yeah that seems very much like it won't always be six lines.

What do the lines you *do* want look like? Can you `print("MARK" + name)` and then `string replace --filter` only the marked lines?

Reply:
> What do the lines you do want look like? Can you print("MARK" + name) and then string replace --filter only the marked lines?

AFAIK they can be any non-empty lines. They can even contain just spaces...

Reply:
Here's what comes up for me:

```
Blender 3.6.0 (hash c7fc78b81ecb built 2023-07-07 13:20:07)
Failed to open dir (No such file or directory): /run/user/1000/gvfs/
Error: Cannot read file "/home/alfa/test.blend": No such file or directory

Blender quit
```

Note that these are very different errors from what you get. It doesn't print anything usable, probably because I don't have any scenes.

Note: The `test.blend` error goes to stderr, the rest goes to stdout.


What you want to test is:

```fish
for name in [scene.name for scene in list(bpy.data.scenes)]:
    print("COMPLETION:", name)' | string replace -rf '^COMPLETION:' ''
```

i.e. instead of the `head` and `tail` guff that, as you see above, isn't usable (because the number of errors differs!), mark the completion line with a string and then only get those.

Reply:
And once you've done that, the order no longer matters and you can remove the `stdbuf`.